src/cljstyle/config.clj

```diff
@@ -6,10 +6,12 @@
   subtree rooted in the directory the file resides in, with deeper files
   merging and overriding their parents."
   (:require
+    [cljstyle.task.print :as p]
```

Comment:
Hmm, I am not too happy about this - other namespaces in `cljstyle.task.*` depend on `cljstyle.config`, so this creates a high-level cycle in the namespace structure. The `ignored?` predicate below already has the config passed into it, could we not add entries to that instead?